fastlane_bot/events/event_gatherer.py

```diff
@@ -30,6 +32,7 @@ def __init__(
             manager: The Manager object
             w3: The connected AsyncWeb3 object.
         """
+        self._blockchain = blockchain
```

Comment:
Can obtain this from `self.cfg.network.NETWORK` or something similar, rather than creating a duplicated attribute.

Reply:
Or maybe `self.cfg.NETWORK`

Reply:
EventGatherer has no access to the config object

Reply:
OK, then it is probably better to add the Config object rather than the blockchain ID, since it ensures that the same blockchain IDs are used across the code (i.e., it helps preventing mistakes, so one just needs to pass the same Config object which is used everywhere else in the system).